profiles/package.mask

```diff
@@ -29,6 +29,14 @@
 
 #--- END OF EXAMPLES ---
 
+# David Seifert <soap@gentoo.org> (2019-07-14)
+# Unmaintained, removed from Debian and other
```

Comment:
What's with the narrow wrapping? You can use up to 80 chars, or 72 if you like ;-P.

Also would be nice to file a central 'treecleaning' bug, so we don't have to go over all of those bugs seeking 'oh no, please don't remove it' comments. Also `Removal in X days`.